layer/decorators/dataset_decorator.py

```diff
@@ -147,20 +148,13 @@ def __call__(self, *args: Any, **kwargs: Any) -> Any:
                     tracker.add_asset(
                         AssetType.DATASET, self.__wrapped__.layer.get_asset_name()
```

Comment:
I don't remember why we chose to use `__wrapped__` (@marta-tatiana or @eminn might chime in), but given that you've removed all but this, wanted to point out.

Reply:
Fixed (in https://github.com/layerai/sdk/pull/137)